android/assets/jsons/translations/template.properties

```diff
@@ -709,6 +712,13 @@ Do you want to exit the game? =
 Start bias: = 
 Avoid [terrain] = 
 
+# Maya calendar popup
+
+The Mayan Long Count = 
+Your scientists and theologians have devised a systematic approach to measuring long time spans - the Long Count. During the festivities whenever the current b'ak'tun ends, a Great Person will join you. = 
+While the rest of the world calls the current year [year], in the Maya Calendar that is: = 
+"[amount] b'ak'tun, [amount] k'atun, [amount] tun = 
```

Comment:
amount, amount2, amount3, otherwise the translation won't work properly

Reply:
:facepalm:  of course